phing/build.yml

```diff
@@ -93,3 +89,5 @@ phpcs:
 project:
   local:
     uri: ${project.local.protocol}://${project.local.hostname}
+  ci:
+    uri: 127.0.0.1:8888
```

Comment:
First of all, thanks for all of your hard work on this. We'll definitely get your work merged in.

But first, I need to get my head around what's happening, so I'll take some notes here for my own usage. On a global level we have:

* project.local.uri
* project.ci.uri
* multisite.names
* site.name
* environment

Then for the active multisite, we generate import variables from the site-specific yaml file into the "site" namespace:

* site.behat...
* site.uri.local
* site.uri.ci
* site.drush.aliases.remote

Couple of things:

1. Why do we use project.ci.uri in one place and site.uri.ci? It seems like we should have a consistent naming scheme.
1. Why import variables into the site namespace at all? It seems simpler to just have the multisite yml file override the global variables, so that, for instance, it changes the values for:
  * project.local.uri
  * project.ci.uri
  * behat.tags
  * drush.aliases.local
  * etc.

I think I may cherry pick some of your code and introduce it in smaller PRs so that we can release it in stages. I'm concerned about too much change too quickly here. I'll get back to you.